### DISCUSSION TITLE
In which path can I find the www pages on the server

### DISCUSSION THREAD

**Discussion by Piratenkanon** [AzuraCast/AzuraCast#6283] [Q&A]
Hello to all,
In which path can I find the www pages on the server
for example the web output from the Icecast server
Thanks in advance

**Comment by robz0** [comment]
I do realise this is not answering your question but 
I kinda had the same question myself while thinking about customising public pages.  If your question has this same intention...

There is this docs page - https://docs.azuracast.com/en/administration/customization - 
Which guides customisation of front page - css, js, video files.
That may be enough for what you need. 

Also, in Stations/Manage, 
there is a panel called Public pages, which shows the URLs for the different public pages of each station - (Front page/On-demand/WebDJ/Podcasts/Schedule)   From there you can use "Embed Widgets", to style and configure widgets that show those  different public pages on any site. 
There are the custom branding options there also.

So am thinking that the way to have your own totally custom page might be to - 
just make your own page however you do that (hand-code, wordpress, some CMS etc) and then 
use Station/Manage/Embed Widgets to get the HTML player embedded on your totally custom web page. (As said, these don't even have to be on the same domain as your AC installation)

More savvy people than I could obviously do much more with CSS/JS customisations but that is what I am thinking. 

Hope that might help in some way.  Depends on your underlying intention for wanting the www path. 

I am only exploring, discovering, testing AzuraCast myself, so complete newbie. But the above is what I have discovered so far and what I am thinking about for customising public pages.  

I do like the default public pages though, which are still very customisable, yet nice clean, simple and unlikely to break with too much direct customisation :)

**Reply by robz0** [reply]
Closer to answering the question maybe?? https://stackoverflow.com/questions/20813486/exploring-docker-containers-file-system
-  i stumbled on this way of looking inside a docker container.  With 'docker export .. ' and other commands on the link you can see the whole innards of the 'azuracast' container.

**Reply by robz0** [reply]
and then...with help from this...
https://stackoverflow.com/questions/30172605/how-do-i-get-into-a-docker-containers-shell
You can get into the running container with something like:
docker exec -it <mycontainer> bash

(Note - the container has to be running)

Then you can run commands like ls, mv etc. There are no doubt other commands that will work but it seems that while you are 'inside' the container, it doesn't recognise commands from the main system - only commands that are setup/defined in the container. When you 'exit' back to the main shell, main system commands/files are available again. 

I was able to access the AzuraCast media folders/file like this. I know it is basic but for me it felt like a victory :)
